File: readability/readability.c
#include <cs50.h>
#include <stdio.h>
#include <ctype.h>
#include <string.h>
#include <math.h>

int main(void)
{

string test = get_string("What string do you want me to test\n");

int letters = 0;
int sentences = 0;
int words = 1;

for (int i = 0; i < strlen(test); i++)
{
    if (isalpha(test[i]) > 0){
        letters++;
    }
    if (test[i] == ' ')
    {
        words++;
    }
    if (test[i] == '.' || test[i] == '!' || test[i] == '?')
    {
        sentences++;
    }
}
    float let = (float) letters / words * 100;
    float sen = (float) sentences / words * 100;
    int score = round( 0.0588 * let - 0.296 * sen - 15.8);

    if (score > 16)
    {
        printf("Grade 16+\n");
    }
    else if (score < 1)
    {
        printf("Before Grade 1\n");
    }
    else
    {
        printf("Grade %i\n", score);
    }
}

File: sentimental-readability/readability.py
from cs50 import get_string

text = get_string("Enter your text: ")

let = 0;
sen = 0;
word = 0;

for i in range(len(text)):
    if(text[i].isalpha()):
        let+=1
    if(text[i] == " "):
        word+=1
    if(text[i] == "." or text[i] == "!" or text[i] == "?"):
        sen+=1

word+=1

L = (let / word) * 100
S = (sen / word) * 100
score = round(0.0588 * L - 0.296 * S -15.8)

if(score > 16):
    print("Grade 16+")
elif(score < 1):
    print("Before Grade 1")
else:
    print("Grade " + str(score))



